search_service/create_database.py

```diff
@@ -43,3 +43,13 @@ def __are_envs_set():
     port = os.getenv("DBAS_PORT", "")
     protocol = os.getenv("DBAS_PROTOCOL", "")
     return "" not in [host, port, protocol]
+
+
+def __check_parameters_set(protocol, host, port):
```

Comment:
Please add the prefix `is_` or `are_` to functions which return a boolean. For example: `are_parameters_empty` or something like this

Reply:
and make the functions speak for themselves. I do not want to look into the implementation to understand what it is "checking"